src/testRunner.js

```diff
@@ -111,6 +111,26 @@ export default class TestRunner {
         });
     }
 
+    /**
+     * Show JSON-formatted metadata listing the tests and their test options,
+     * tags, etc.
+     *
+     * @param {RegExp} filter
+     */
+    testsMeta(filter) {
+        const tests = this.getTests(filter, { logSkippedTests: false });
+        const testNames = tests.map(test => test.name);
+        const metadataByName =
+            tests.map(test => ({...test, filename: options.getFilename(test)}))
+                 .map(({name, testOpts, suite, filename}) => ({name, testOpts, suite, filename}))
+                 .reduce((obj, test) => (obj[test.name] = test, obj), {});
+        const metadata = {
+            "by_test_name": metadataByName,
+            "order": testNames
+        }
+        console.log(JSON.stringify(metadata))
```

Comment:
Could this method _return_ `metadata`, then stringify and print the value in core? Would give a bit more flexibility if we wanted to use this elsewhere.

If so, could rename to something like `getTestsMeta` instead

Reply:
Good idea. I think that should be addressed now, in conjunction with this change in my corresponding platform PR https://github.com/IQGeo/myworld-product-core/pull/2011/commits/975e9664dc29839da07bfdf961d6f05a89c10fab